mod/profile/views/default/profile/layout.php

```diff
@@ -1,13 +1,7 @@
 <?php
 /**
  * Profile layout
- * 
- * @uses $vars['entity']  The user
  */
 
-// main profile page
-$params = array(
-	'content' => elgg_view('profile/wrapper'),
-	'num_columns' => 3,
-);
-echo elgg_view_layout('widgets', $params);
+echo elgg_view('profile/wrapper');
+echo elgg_view_layout('widgets', ['num_columns' => 2]);
```

Comment:
Pass owner explicitly, chain from resource view if needed. Enough of this page owner magic.